documentation/modules/ROOT/pages/_partials/build-containers.adoc

```diff
@@ -19,7 +19,7 @@ After successful maven build, run the following command to build te container:
 [#conk8s-build-quarkus-jvm-app-container]
 [source,bash,subs="+macros,+attributes"]
 ----
-docker build -f src/main/docker/Dockerfile.jvm -t example/{quarkus-project-name}-jvm .
+docker build -f src/main/docker/Dockerfile.jvm -t example/{quarkus-project-name}:1.0-SNAPSHOT .
```

Comment:
instead of `1.0-SNAPSHOT` shall we can use `1.0-SNAPSHOT-jvm` that gives a better clarity of what image the users have built

Reply:
Well actually there is a problem on that and it is that dekorate uses the artifact version to generate the resources and set the container image. Until this is not configurable if we change the version of the container to something different than the artifact version then we cannot use dekorate (or we need to ask the attendee to change the autogenerated file) which is a new step that user might not do it correctly.

Reply:
we need to think on it as doing so we are teach something wrong as we try to push two different formats of the container under same tag

Reply:
yeah I know, I am asking if there is any way to do it. Although I agree with you about this, also it is true that in the workshop context, probably everyone will choose only one way (my bet is JVM for compilation time thing) so let's say that it is not terrible at all and also helps to avoid any future problem. For example in a previous step user has created in jvm mode, then in native mode, but when it is the time to deploy the example instead of setting the native one, he copies the jvm mode, so nothing goes wrong as both containers exist, but he will not see the expected result (as the container is an old one) and he might get crazy on checking why it is not working. If the container is named in all cases equally, then there is an override policy so they get always the latest one. As I said not the best in "prod" thing bu might help an end user to not do any error of this kind.